pkg/proxy/proxy.go

```diff
@@ -457,6 +457,12 @@ func (p *Proxy) RemoveRedirect(id string, wg *completion.WaitGroup) error {
 	go func() {
 		time.Sleep(portReleaseDelay)
```

Comment:
This introduces the requirement that `portReleaseDelay >= 2 * proxyConnectionCloseTimeout`, to leave the time to the proxy to cleanly close both connections. It is the case now, but I think that should be documented on both `portReleaseDelay` and `proxyConnectionCloseTimeout` so that requirement is met when either constant is modified.

Reply:
We should enforce `portReleaseDelay >= 2 * proxyConnectionCloseTimeout` to be on the safe side

Reply:
Agreed. How about defining it that way?
```
portReleaseDelay = 2 * proxyConnectionCloseTimeout
```

Reply:
^ Should we document this rationale in comments while defining ? That portReleaseDelay is twice of proxyConnectionCloseTimeout